.rubocop-rs-config.yml

```diff
@@ -0,0 +1,187 @@
+Layout/ClassStructure:
+  Enabled: true
```

Comment:
just curious, but why do we need to specify that these rules are enabled? is it because they are disabled by default?